requirements_lambda.txt

```diff
@@ -5,4 +5,4 @@ numpy==1.26.4
 sentence-transformers==3.4.1
 sentry-sdk==2.8.0
 toml==0.10.2
-torch==2.2.2+cpu
+torch==2.2.2
```

Comment:
The lambda uses the cpu only version as it's a much smaller library so we need to keep that in

Reply:
So the plan for this requirements_lambda.txt was basically to isolate the smaller subset of dependencies we needed in the lambda image. Generally it shouldn't really be getting used locally I don't think (unless you're building the docker container)

Reply:
So you ought to be able to revert this change and just do:

```
pip install --extra-index-url https://download.pytorch.org/whl/cpu -r requirements.txt
```

Reply:
Before/after screenshots are the same with and without +cpu so I assume the upstream now handles this better based on detected hardware ![screenshot-2025-02-17_11-51-19](https://github.com/user-attachments/assets/a2f73f7d-ff42-417e-8018-a3f8f65465a6)